docs/source/adding-yara-rules.rst

```diff
@@ -12,22 +12,74 @@ BinaryAlert includes a number of `custom YARA rules <https://github.com/airbnb/b
 
 Clone Rules From Other Projects
 -------------------------------
-BinaryAlert makes it easy to clone YARA rules from other open-source projects:
+BinaryAlert makes it easy to clone YARA rules from other projects:
 
 .. code-block:: bash
 
   $ ./manage.py clone_rules
 
-This will copy a subset of YARA rules from several :ref:`open-source collections <yara-credits>`.
-You can add more rule sources in `rules/clone_rules.py <https://github.com/airbnb/binaryalert/blob/master/rules/clone_rules.py>`_
+This will copy a subset of YARA rules from several default :ref:`open-source collections <yara-credits>` into the ``rules/`` folder.
+The cloned folder structure will mirror that of the remote repository.
 
-.. note:: We are working on a more expressive configuration for cloning subsets of rule repositories.
+.. note:: To ensure all upstream changes are copied (including deletions), the cloned folder structure for each repo will be deleted before cloning. For example, ``rules/github.com/Yara-Rules/rules.git`` will be deleted from your local filesystem before cloning from ``Yara-Rules``.
 
+Configuring Rule Sources
+........................
+
+You can configure the remote rule sources in `rules/rule_sources.json <https://github.com/airbnb/binaryalert/blob/master/rules/rule_sources.json>`_. Each rule source is defined by a git-cloneable ``url``, an optional list of filepaths to ``include``, and an optional list of filepaths to ``exclude``.
+
+Some examples using the `Yara-Rules <https://github.com/Yara-Rules/rules>`_ repository:
+
+**1. URL only**
+
+.. code-block:: json
+
+    {
+      "repos": [
+        {
+          "url": "https://github.com/Yara-Rules/rules.git"
+        }
+      ]
+    }
+
+If you specify just the ``git`` URL, BinaryAlert will traverse the entire repo and copy every ``.yar`` and ``.yara`` file (case insensitive).
+SSH URLs (e.g. ``git@github.com:Yara-Rules/rules.git``) are also supported, since BinaryAlert just runs a ``git clone`` on the specified URL.
+
+**2. Filter with Include and Exclude**
+
+The ``Yara-Rules`` repo is very large, and you may only be interested in a specific subset of rules:
+
+.. code-block:: json
+
+    {
+      "repos": [
+        {
+          "url": "https://github.com/Yara-Rules/rules.git",
+          "include": [
+            "CVE_Rules/*",
+            "Malware/*"
+          ],
+          "exclude": [
+            "Malware/POS*",
+            "*_index.yar"
+          ]
+        }
+      ]
+    }
+
+This will copy rules from the ``CVE_Rules`` and ``Malware`` folders, excluding POS and index files. (This example is just for demonstrative purposes and isn't necessarily recommended.) BinaryAlert runs Unix filename pattern matching via `fnmatch <https://docs.python.org/3.6/library/fnmatch.html>`_.
```

Comment:
Can you change the stuff in parenthesis: `(This example...` to a note using the `.. note:: ` directing in RST. That way it will get highlighted better and not overlooked.

Reply:
:+1: